doc/development_guide/contribute.rst

```diff
@@ -160,6 +160,12 @@ current environment in order to have faster feedback. Run from Pylint root direc
 
     python tests/test_functional.py
 
+You can use all the options you would use for pytest, for example `-k "test_functional[len_checks]"`.
+This also permit to automatically update the expected output instead of checking it with the
```

Comment:
Why not:
It is also possible to update the expected output instead of using it for comparison, by appending the `--update-functional-output` to the command line::

Reply:
Done :)